src/libs/actions/ReimbursementAccount/index.ts

```diff
@@ -11,26 +13,20 @@ export {setBankAccountFormValidationErrors, setPersonalBankAccountFormValidation
  * - `null` if we want to go back to the view where the user selects between connecting via Plaid or connecting manually
  * - CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL to ask them to enter their accountNumber and routingNumber
  * - CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID to ask them to login to their bank via Plaid
- *
- * @param {String | null} subStep
- * @returns {Promise<void>}
  */
-function setBankAccountSubStep(subStep) {
+function setBankAccountSubStep(subStep: BankAccountSubStep | null) {
     return Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {achData: {subStep}});
 }
 
 function hideBankAccountErrors() {
     Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {error: '', errors: null});
 }
 
-function setWorkspaceIDForReimbursementAccount(workspaceID) {
+function setWorkspaceIDForReimbursementAccount(workspaceID: string | null) {
```

Comment:
I search for a usage for this function and it seems its not used anywhere, maybe we can delete it?

Reply:
@kubabutkiewicz  there are even more functions that are not used anywhere or are only re-exported in other modules but again not used. I'm not sure if this is a mistake and they should be deleted or temporarily those functions are not in use. cc: @VickyStash 

Here's the full list of what I found:

Not used anywhere:

`getReimbursementAccountInSetup`
`getReimbursementAccountWorkspaceID`
`getCredentials`

Exported in `ReimbursementAccount` (`index.ts`) but not used anywhere:

`setPersonalBankAccountFormValidationErrorFields`
`showBankAccountFormValidationError`
`deleteFromBankAccountList`

Exported in `BankAccount.ts` and `ReimbursementAccount` (`index.ts`) but not used anywhere:

`setBankAccountFormValidationErrors`
`setWorkspaceIDForReimbursementAccount`

Reply:
@fabioh8010 do you think its safe to remove them?

Reply:
I think we could check if he had recent activity on these functions, because sometimes we have code that looks unused but it was created with the purpose to be used later (during a feature development for example).

Reply:
Well, those functions seems to be pretty old:

`getReimbursementAccountInSetup` (~2 years) https://github.com/Expensify/App/pull/6230
`getReimbursementAccountWorkspaceID` (~2 years) https://github.com/Expensify/App/pull/6230
`getCredentials` (~2 years) https://github.com/Expensify/App/pull/6230
`setPersonalBankAccountFormValidationErrorFields` (~20 months) https://github.com/Expensify/App/pull/9423
`showBankAccountFormValidationError` (~2 years) https://github.com/Expensify/App/pull/6230
`deleteFromBankAccountList` (~2 years) https://github.com/Expensify/App/pull/7557
`setWorkspaceIDForReimbursementAccount` (~2 years) https://github.com/Expensify/App/pull/6230

Reply:
Thanks for the info @mateuuszzzzz ! Well, I suppose we can remove them, let's do it and see how it goes during C+/Exfy reviews.

Reply:
Cool, I removed them